README.md

```diff
@@ -1 +1,2 @@
-# docker-vault-init
\ No newline at end of file
+# Docker Vault Init
```

Comment:
I think the `README` needs some instructions just so that someone who's new to the project can get up to speed without having to figure out a lot by looking at the code. Something like (to be modified appropriately) ...

```markdown
## Development

Below instructions are only necessary if you intend to work on the source code.
For normal usage the above installation instruction should do.

### Requirements

1. Ensure that you have a [properly
   configured](https://golang.org/doc/code.html#Workspaces) Go workspace.
1. For dependency management, you'll need `dep` which can be installed with
   `brew install dep`.

### Building

1. Fetch the code with `go get -v github.com/Intellection/docker-vault-init`.
1. Install the Go development tools via `make tools`.
1. Install `dep` via `brew install dep`.
1. Install application dependencies via `make dependencies` (they'll be placed
   in `./vendor`). Requires [golang/dep][dep] package manager.
1. Build and install the binary with `make build tag=x.y.z`.
1. Run the command e.g. `./bin/vault-init help` as a basic test.

### Testing

1. Install the Go testing tools via `make tools`.
2. Run linter using `make lint` and test using `make test`.
```

Reply:
Updated with suggested dev info here https://github.com/Intellection/vault-init/pull/1/commits/58e355dd2cf8c3be23b72961de69c3a80bdd57f1